### PR TITLE
Added OGC resource server deployment stack files

### PR DESCRIPTION
**Pull request by sivanaikk**

### CODE REVIEW

File: Docker-Swarm-deployment/single-node/ogc-rs/example-secrets/secrets/config.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+	"version": "1.0",
+	"zookeepers": [
+		"zookeeper.zookeeper.svc.cluster.local"
```
[the remark this answers is not in the document]
Comment (sivanaikk) [reply Docker-Swarm-deployment/single-node/ogc-rs/example-secrets/secrets/config.json]: modified as suggested!

File: Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+
+services:
+  file-server:
```
Comment (pranavv0) [inline Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml:4]: change the service name to ogc-rs

File: Docker-Swarm-deployment/single-node/ogc-rs/example-secrets/secrets/config.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+			"s3BucketName": "<S3 BucketName>",
+                        "s3Region": "<S3 Bucket Region>",
+                        "s3AccessKey": "<S3 AccessKey>",
+                        "s3SecretKey": "<S3 SecretKey>"
+		},
+		{
+			"id": "ogc.rs.authenticator.AuthenticationVerticle",
+			"isWorkerVerticle": false,
+			"verticleInstances": 1,
+			"audience": "ogc.iudx.io",
```
Comment (abhi4578) [inline Docker-Swarm-deployment/single-node/ogc-rs/example-secrets/secrets/config.json:28]: Add placeholder value.
``<ogc server audience Ex:ogc.iudx.io>``

File: Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    image: ghcr.io/datakaveri/geoserver-dev:1.0.0-alpha-96bb4be
+    command: bash -c "exec java $$OGCRS_JAVA_OPTS  -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.Log4j2LogDelegateFactory -jar ./fatjar.jar  --host $$(hostname) -c secrets/configs/config.json"
+    cap_drop:
+      - ALL
+    read_only: true
+    env_file:
+      - ./secrets/.ogc-rs.env
+    volumes:
+      - type: volume
+        source: ogc-rs-volume
```
Comment (abhi4578) [inline Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml:14]: Is this volume needed? 
cc + @karun-singh

**Review by abhi4578 — CHANGES_REQUESTED** [review]

File: Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml (hunk 4 above)
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml:14]: @abhi4578 @sivanaikk No, this is not needed. Can be removed.

Comment (sivanaikk) [reply Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml]: Updated the stack file

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**